### DISCUSSION TITLE
Supporting Legacy Applications

### DISCUSSION THREAD

**Discussion by justinMBullard** [rokucommunity/brighterscript#572] [Ideas]
@TwitchBronBron  It would be nice to have an option to force BrighterScript compile for .brs files.  I have legacy app that I want to leverage BrighterScript functionality, but switching extensions would add a level of unwanted complexity.

I have a branch that add forceBrighterScript to the bsconfig.json file. If you think this would be a good idea, it would be glad to share.

**Comment by TwitchBronBron** [comment]
I'm definitely open to having this flag be added. Maybe call it `transpileBrs`?

**Reply by justinMBullard** [reply]
@TwitchBronBron I will make the name change and then push the branch.

**Reply by justinMBullard** [reply]
@TwitchBronBron I tried to push my branch but it complained. 
```
ERROR: Permission to rokucommunity/brighterscript.git denied to justinMBullard.
```

**Reply by TwitchBronBron** [reply]
You need to make a fork of the repo that you control, push your branch there, and then you can open a PR.
